src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -311,6 +311,8 @@ export class ContextualMenu extends React.Component<IContextualMenuProps, IConte
       if (!item.items || !item.items.length) { // This is an item without a menu. Click it.
         if (item.onClick) {
           item.onClick(item, ev);
+        } else if (item.href) {
+          location.href = item.href;
```

Comment:
hmm this is not good. If we're taking an HREF, we should render a hyperlink. location.href assignments don't respect normal link behaviors, such as middle button clicks, ctrl clicking to open in new tab, etc. Plus screen readers can identify their purpose a bit better.

A better fix would be that if we support HREFs, we render them as hyperlinks that look like buttons. This is what we do in both Link and Button; if you render a Link with no href it renders as a button. If you render a Button with an href, it renders it using an A tag.